javaSE/src/main/java/com/smartdevicelink/managers/lifecycle/LifecycleManager.java

```diff
@@ -651,7 +678,7 @@ public void onProtocolServiceDataACK(SessionType sessionType, int dataSize, byte
 
     @Override
     public void onAuthTokenReceived(String token, byte sessionID) {
```

Comment:
This method should be called in another class to set the token. I think the value should be set in SdlProtocol then propagated to LifecycleManager